VirtoCommerce.Domain/Pricing/Model/PriceCalendarChange.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Domain.Pricing.Model
```

Comment:
I think this type is redundant because contains single property of the product identifier which can be used as it without the special type

Reply:
This is for future proofing, so we can easily add more data without breaking the interface.